### ISSUE TITLE
[Bug] Extension readcomiconline seems to run into a captcha

### ISSUE THREAD

**Issue by Dorifer** [AriaMoradi/Tachidesk#271]
## Device information
- Tachidesk version: Server: v0.6.0 + WebUI: r893 tested in the docker image and natively on debian
- Server Operating System: debian stable
- Server Desktop Environment: virtual server
- Server JVM version: docker image
- Client Operating System: Manjaro Linux
- Client Web Browser: Firefox

## Steps to reproduce
1. install readcomiconline extension
2. open any chapter of any comic

### Expected behavior
the comic should open

### Actual behavior
in the browser just a blank, white page is shown. 

output of the debug shell: 
`INFO: <-- 200 https://readcomiconline.li/Special/AreYouHuman?reUrl=%2fComic%2fChu%2fIssue-7%3fid%3d187626%26quality%3dhq (977ms, unknown-length body`

## Other details
tachidesk seems to trigger a captcha when downloading the pages for a comic on readcomiconline. Weirdly this does not happen in tachidesk-jui with the internal server or the android app. I tried re-downloading the docker image and also started the electron Version directly on the server with debug on.

---

**Comment by AriaMoradi** [comment]
It might be that your ip is flagged as used in bots

**Comment by Syer10** [comment]
The ReadComicOnline source is broken on Tachiyomi's side currently

**Comment by Syer10** [comment]
ReadComicOnline uses QuickJS, which only works in specific senerios. 
1. Windows is not supported by QuickJS
2. Linux users need to find a way to install libc++.so.1, on Ubuntu do `sudo apt install libc++-dev`, on Fedora do `sudo dnf install libcxx`
3. I haven't tested MacOS yet, not sure if I will currently
4. Adding support for it in the docker container is in the works here https://github.com/Suwayomi/docker-tachidesk/pull/25

**Comment by AriaMoradi** [comment]
Fixed in v0.6.6

**Comment by barrelltitor** [comment]
Happening again, but some new logs:

```
WARN  io.javalin.Javalin - Uncaught exception
java.lang.Exception: Unable to de-obfuscate rguard script

```
Before this, I also got

```
[DefaultDispatcher-worker-7] [34mINFO [0;39m suwayomi.tachidesk.manga.impl.download.Downloader - Downloader faced an exception
java.lang.Exception: Solve captcha in WebView
```
Seems to have been fixed recently in main tachiyomi: https://github.com/tachiyomiorg/tachiyomi-extensions/pull/16900
